index.js

```diff
@@ -60,7 +60,7 @@ function Hook (modules, options, onrequire) {
 
     const filename = Module._resolveFilename(id, this)
     const core = isCore(filename)
-    let moduleName, basedir
+    let moduleName, basedir, absoluteRequire
```

Comment:
I prefer if we explicitly set `absoluteRequire` to `false` as the default value. That signals intent better and helps V8 😃